<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Almuerzo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Módulo 3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Módulo 5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Martes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Miércoles</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Jueves</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Módulo 6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Módulo 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lunes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Módulo 2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Módulo 4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Viernes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBox_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBox_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="checkBox_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="checkBox_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QCheckBox" name="checkBox_14">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBox_15">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBox_16">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="checkBox_17">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="checkBox_18">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="checkBox_19">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="checkBox_20">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QCheckBox" name="checkBox_21">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="checkBox_22">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="checkBox_23">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="checkBox_24">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QCheckBox" name="checkBox_25">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QCheckBox" name="checkBox_26">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QCheckBox" name="checkBox_27">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QCheckBox" name="checkBox_28">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="checkBox_29">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="checkBox_30">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="checkBox_31">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QCheckBox" name="checkBox_32">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QCheckBox" name="checkBox_33">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QCheckBox" name="checkBox_34">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QCheckBox" name="checkBox_35">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar módulos disponibles</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>89</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="warning_message">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>200</number>
     </property>
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="text">
        <string>Atrás</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
